specification/storage/data-plane/Microsoft.StorageDataLake/stable/2019-12-12/DataLakeStorage.json

```diff
@@ -2213,6 +2235,13 @@
             "required": false,
             "type": "string"
           },
+          {
+            "name": "paginated",
+            "in": "query",
+            "description": "Optional and valid only if Hierarchical Namespace is enabled for the account and resource is a directory with \"recursive\" query parameter set to \"true\". For recursive directory deletion, the number of paths that could be deleted with each invocation is limited when the authorization mechanism used is ACL and the caller is a non-super user, as the default timeout is 30 seconds. When \"paginated\" query parameter is set to \"true\", the response header may contain \"x-ms-continuation\" if the above limit is hit. While sending the \"x-ms-continuation\" in the subsequnt request, \"paginated\" and \"recursive\" query parameter should be set to \"true\". When the response doesnt contain any \"x-ms-continuation\", recursive directory deletion is successful. The actual directory deletion happens only in the last invocation, the previous ones involve ACL checks in the server of the files and directories under the directory to be recursively deleted.",
```

Comment:
`When the response doesnt contain any \"x-ms-continuation\"` -> `When the response doesn't contain a \"x-ms-continuation\" header.`?